includes/reader-activation/sync/class-metadata.php

```diff
@@ -0,0 +1,397 @@
+<?php
+/**
+ * Reader Activation Sync Metadata.
+ *
+ * @package Newspack
+ */
+
+namespace Newspack\Reader_Activation\Sync;
+
+use Newspack\Reader_Activation\Sync;
+use Newspack\Reader_Activation\ESP_Sync;
```

Comment:
Nit: unused `use`

Reply:
Removed in d22dd80cf82f03ebe88b42d7f811cc7a0eacaefc